chutney/server/src/main/java/fr/enedis/chutney/execution/domain/campaign/CampaignExecutionEngine.java

```diff
@@ -108,31 +109,31 @@ public List<CampaignExecution> executeByName(String campaignName, String environ
         List<Campaign> campaigns = campaignRepository.findByName(campaignName);
         return campaigns.stream()
             .map(campaign -> selectExecutionEnvironment(campaign, environment))
-            .map(campaign -> executeScenarioInCampaign(campaign, userId, dataset))
+            .map(campaign -> executeScenarioInCampaign(campaign, userId, dataset, null))
```

Comment:
keep `executeScenarioInCampaign(campaign, userId, dataset)` that internally calls `executeScenarioInCampaign(campaign, userId, dataset, null)`.